camelComponent/src/test/java/io/vantiq/extsrc/camel/VantiqComponentTest.java

```diff
@@ -255,6 +268,93 @@ public void testVantiqConsumer() throws Exception {
         assert uniqueMsgs.size() == expectedMsgCount;
     }
     
+    @Test
```

Comment:
Duplicates the other tests but verifies that we have JSON coming thru.  And adds a few other types to test